docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -111,32 +186,37 @@ You use components in the `config.alloy` file to tell {{< param "PRODUCT_NAME" >
 The examples run on a single host so that you can run them on your laptop or in a Virtual Machine.
 You can try the examples using a `config.alloy` file and experiment with the examples.
 
-### First component: Log files
-
-Create a file called `config.alloy` in your current working directory and paste the following component configuration at the top of the file:
+### Create a `config.alloy` file
 
-```alloy
-local.file_match "local_files" {
-    path_targets = [{"__path__" = "/var/log/*.log"}]
-    sync_period = "5s"
-}
+To start create a `config.alloy` file within your current working directory:
```

Comment:
Hi @clayton-cornell, so I changed the docker compose command to:
```
 docker compose up -d
```
This will run the docker compose in detached mode which will free the terminal up to allow other commands to be run